user_manual_de/files/encrypting_files.rst

```diff
@@ -1,143 +1,146 @@
-administrator all of your Nextcloud data files are automatically encrypted. 
-Encryption is server-wide, so when it is enabled you cannot choose to keep your 
-files unencrypted. You don't have to do anything special, as it uses your 
-Nextcloud login as the password for your unique private encryption key. Just log 
-in and out and manage and share your files as you normally do, and you can 
-still change your password whenever you want.
-
-Its main purpose is to encrypt files on remote storage services that are 
-connected to your Nextcloud server, such as Dropbox and Google Drive. This is an 
-easy and seamless way to protect your files on remote storage. You can share 
-your remote files through Nextcloud in the usual way, however you cannot share 
-your encrypted files directly from Dropbox, Google Drive, or whatever remote 
-service you are using, because the encryption keys are stored on your Nextcloud 
-server, and are never exposed to outside service providers. 
-
-If your Nextcloud server is not connected to any remote storage services, then 
-it is better to use some other form of encryption such as file-level or whole 
-disk encryption. Because the keys are kept on your Nextcloud server, it is 
-possible for your Nextcloud admin to snoop in your files, and if the server is 
-compromised the intruder may get access to your files. (Read 
-`How Nextcloud uses encryption to protect your data  
-<https://owncloud.org/blog/how-owncloud-uses-encryption-to-protect-your-data/>`_ 
-to learn more.)
-
-Encryption FAQ
---------------
-
-How Can Encryption Be Disabled? 
-^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
-
-The only way to disable encryption is to run the :ref:`"decrypt all" <encryption_label>` 
-script, which decrypts all files and disables encryption.
-
-Is It Possible To Disable Encryption With The Recovery Key?
-^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
-
-Yes, *if* every user uses the :ref:`file recovery key
-<enable-file-recovery-key>`, :ref:`"decrypt all" <encryption_label>` will use it
-to decrypt all files.
-
-Can Encryption Be Disabled Without The User’s Password?
-^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
-
-If you don't have the users password or :ref:`file recovery key
-<enable-file-recovery-key>` then there is no way to decrypt all files. What's
-more, running it on login would be dangerous, because you would most likely run
-into timeouts. 
-
-Is It Planned To Move This To The Next User Login Or A Background Job? 
-^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
-
-If we did that, then we would need to store your login password in the database.
-This could be seen as a security issue, so nothing like that is planned.
+======================================
+Verschlüsseln Ihrer Daten in Nextcloud
+======================================
+
+Nextcloud enthält eine Verschlüsselungs-App, die, sobald durch Ihren Administrator
```

Comment:
Die App heißt aber nicht "Verschlüsselungs", deshalb nicht in Anführungszeichen.